e2e/transaction_test.go

```diff
@@ -205,8 +205,14 @@ func TestEthTransfer(t *testing.T) {
 			ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 			defer cancel()
 			receipt, err := srv.WaitForReceipt(ctx, txnHash)
-			assert.NoError(t, err)
-			assert.NotNil(t, receipt)
+
+			if testCase.shouldSuccess {
```

Comment:
I think we have this typo in other tests as well, and it just got propagated.
This should be `shouldSucceed` instead of `shouldSuccess`

Reply:
👍 https://github.com/0xPolygon/polygon-sdk/pull/103/commits/d1cf11131d621b758cadb8ec2ee18fc13b3cb630